meta-mender-qemu/docker/qemux86-64/Dockerfile

```diff
@@ -51,4 +51,5 @@ RUN if [ `stat -c %s ovmf.qcow2` -eq 0 ]; then rm -f ovmf.qcow2; fi
 
 RUN chmod +x entrypoint.sh mender-qemu
 EXPOSE 8822
+EXPOSE 85
```

Comment:
There is no port 80 here! [This](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-qemu/scripts/docker/entrypoint.sh#L41) is what needs to change.